gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -840,38 +840,40 @@ function TellTraitorsAboutTraitors()
 			traitornicks[#traitornicks + 1] = v:Nick()
 		end
 	end
+	
+	for _, v in ipairs(player.GetAll()) do
+		if v:HasTeam(TEAM_TRAITOR) then
 
-	hook.Run("TTT2TellTraitors", traitornicks)
+			local tmp, shouldShow = hook.Run("TTT2TellTraitors", v, table.Copy(traitornicks))
 
-	-- This is ugly as hell, but it's kinda nice to filter out the names of the
-	-- traitors themselves in the messages to them
-	local traitornicks_min = #traitornicks < 2
+			if shouldShow == nil then shouldShow = true end
```

Comment:
simplify with `shouldShow = shouldShow or true`

Reply:
when I do that, it somehow always returns true.

Reply:
what do you mean? ^^

Reply:
If I use `shouldShow = shouldShow or true` instead of my thing above, then It will always get back to true, even if I return false in the hook.

Reply:
Sure, I did too much C#, or just checks for "not nil or not false"... Just focus on the following comment ^^

Reply:
Made changes, please look at them, is that what you meant ?
https://github.com/TTT-2/TTT2/pull/361/commits/0008e2a4c86f48eb4a1780de7946edeb6e8feb0e